app/models/patron.rb

```diff
@@ -15,10 +40,18 @@ def initialize(record)
     @record = record
   end
 
+  def to_param
+    id
```

Comment:
mainly here so that we have a not-ugly patron path (mine would be /patron/cdm32) though the actual id param is irrelevant (could be literally anything)

Reply:
makes sense